controllers/tempest_controller.go

```diff
@@ -424,6 +425,8 @@ func (r *TempestReconciler) generateServiceConfigMaps(
 	templateParameters["ImageRefAlt"] = "image_alt_uuid"
 	templateParameters["IdentityUriV3"] = "http://keystone-public-openstack.apps-crc.testing"
 	templateParameters["AdminPassword"] = "adminpass"
```

Comment:
do we still need AdminPassword and IdentityUriV3 now when we have what You have added below?

Reply:
I left that for now, I'm thinking in the future to add an option if the person want's to use the built-in tempestconf from the tempest container, or if they want to skip it and pass all the info manually. What do you think?

Reply:
it was there just as a first step I think but we should rely on the info provided in the CR's yaml file I think. Maybe we can add some default values for it in the API definition but I don't think we should hard code it here

Reply:
Ok, I'll remve the whole config and the tempest.conf since it's not being used.